src/ShortcutGuide.tsx

```diff
@@ -22,7 +22,7 @@ const ShortcutGuide: React.FC = () => {
 
 	return (
 		<>
-			<div className="a11y_modal" data-testid="a11y_modal" role="button">
+			<div className="a11y_modal" data-testId="a11y_modal" role="button">
 				<div
 					className="a11y_modal_content"
 					aria-label="AutoVizually shortcut guide. AutoVizually lets you navigate between
```

Comment:
This could be a component prop with a default value.

Also, I believe it makes more sense to move this label into a table caption (even if visually hidden) rather that having it here, on an element that does not receive focus and is, in fact, a generic div element, with no semantic whatsoever.